src/main/java/com/puppycrawl/tools/checkstyle/AuditEventDefaultFormatter.java

```diff
@@ -55,9 +52,7 @@ public String format(AuditEvent event) {
             severityLevelName = severityLevel.getName().toUpperCase(Locale.US);
         }
 
-        // Avoid StringBuffer.expandCapacity
-        final int bufLen = calculateBufferLength(event, severityLevelName.length());
-        final StringBuilder sb = new StringBuilder(bufLen);
+        final StringBuilder sb = new StringBuilder();
```

Comment:
this is perofrmance optimization.
Lets add this method to pom.xml for pitest to avoid.
Method of this method is pretty uniq, it is good.